docs/docs/dev/installation/docker.md

```diff
@@ -78,9 +78,9 @@ mtls:
 
 You can run the following script to automatically detect the operating system and download Kuma:
 
-```sh
-curl -L https://kuma.io/installer.sh | sh -
-```
+<div class="language-sh">
+<pre><code>curl -L https://kuma.io/installer.sh | VERSION={{ $page.latestVersion }} sh -</code></pre>
+</div>
```

Comment:
**Suggestion**: This change should be undone because this is the dev (or latest) documentation for which an explicit version number should explicitly not be set.

~~~suggestion
```sh
curl -L https://kuma.io/installer.sh | sh -
```
~~~